lua/typescript-tools/autocommands/jsx_close_tag.lua

```diff
@@ -0,0 +1,83 @@
+local api = vim.api
+
+local common = require "typescript-tools.autocommands.common"
+local plugin_config = require "typescript-tools.config"
+local plugin_api = require "typescript-tools.api"
+
+local M = {}
+
+--- @generic T
+--- @param list T[]
+--- @param value T
+--- @return boolean
+local function list_contains(list, value)
+  if vim.list_contains ~= nil then
+    return vim.list_contains(list, value)
+  end
+  for _, v in ipairs(list) do
+    if value == v then
+      return true
+    end
+  end
+
+  return false
+end
```

Comment:
This may land into:
https://github.com/pmizio/typescript-tools.nvim/blob/master/lua/typescript-tools/utils.lua

Reply:
moved  https://github.com/pmizio/typescript-tools.nvim/pull/185/commits/77fbf442693c6d359c351d5968922a336fe6869d